tsconfig.json

```diff
@@ -21,6 +21,7 @@
 		"node_modules",
 		"build",
 		"src/**/*.stories.tsx",
-		"src/**/*.test.tsx"
+		"src/**/*.test.tsx",
+		"src/**/*.test.ts"

```

Comment:
Needed to add this to prevent tests from being published, because "Why would you?".